src/pymor/operators/fv.py

```diff
@@ -165,7 +165,7 @@ def __init__(self, flux, flux_derivative, gausspoints=5, intervals=1):
         self.intervals = intervals
         self.build_parameter_type(inherits=(flux, flux_derivative))
         points, weights = GaussQuadratures.quadrature(npoints=self.gausspoints)
-        points = points / intervals
+        points /= intervals
```

Comment:
points is write-protected. The original version makes a copy of points, the new in-place version will fail.